CHANGELOG.TXT

```diff
@@ -13,6 +13,66 @@ specified here
 						added			/lib/models/report which contains report definiton/generation objects
 						added			/templates which contains all UI templates
 
+New & Changed Features for OrangeHRM ver 4.1.2
+    New Features
+    ------------
+        Enhancement of system configuration wizard during installation. Setup instance with user entered data during installation.
+        Supported language packs
+        -English
+        -French
+        -German
+        -Spanish (Costa Rica)
+        -Spanish
+        -Dutch
+        -Chinese (Traditional Han)
+        -Chinese (Simplified Han)
+    Support of cli installer
+
+    Bug Fixes
+    ------------
+    "Csrf token validation failed" message appears in login screen even after trying to login with valid credentials git issue #247
+    Incorrect version number in config file git issue #262
+    Cannot install in mysql 5.7.23, supporting version is equal or lesser version than 5.7.22
+    Web installer: "Help" link should be changed with a new guide
+    When change password by accessing change password link, system getting inaccessible

```

Comment:
System becomes inaccessible after changing password by accessing the change password link

Reply:
Done.